### DISCUSSION TITLE
install: so-test fail

### DISCUSSION THREAD

**Discussion by NavikaG** [Security-Onion-Solutions/securityonion#3779] [Unsupported Versions]
So I have been working on this for a while and I cant get my install to work i'm getting this error and I just cant figure out why. Im trying to install on virtual box on a Ubuntu system

![image](https://user-images.githubusercontent.com/32896308/113811825-c634dc00-9721-11eb-8610-c71f31bb4e5e.png)

**Comment by jertel** [comment]
Review your `/opt/so/log/tcpreplay/init.log`.
